test/extended/util/client.go

```diff
@@ -519,10 +519,6 @@ func (c *CLI) AdminQuotaClient() quotav1client.Interface {
 	return quotav1client.NewForConfigOrDie(c.AdminConfig())
 }
 
-func (c *CLI) AdminOAuthClient() oauthv1client.Interface {
```

Comment:
@stlaz which of `AdminOAuthClient` and `AdminOauthClient` is the right name for the method?

Reply:
Original name was `AdminOauthClient`, the other one was added later and the code for both functions was exactly the same. Thought I should unify it.

Reply:
@pacevedom let's do `AdminOAuthClient`.

Reply:
Done